src/dashboard/helper/validation.tsx

```diff
@@ -0,0 +1,26 @@
+export function validateUsername(value: string) {
+  // Check if the username is at least 4 characters long and does not contain any special characters
+  if (value.length < 4) {
+    return 'Name must have at least 4 letters';
+  } else if (!/^[a-zA-Z0-9_]*$/.test(value)) {
+    return 'Name must not contain any special characters';
+  }
+  return null;
+}
+
+export function validatePassword(value: string) {
```

Comment:
bruh i hate these kind of password systems

Reply:
You mean 2 factor auth?

Reply:
nah i wanna do the login via gmail, facebook etc. (2FA is also another good feature, added to the pipeline)

Reply:
Oh gotcha yeah that would be good could potentially also look into https://webauthn.io/ which is way better and way easier if u have a fingerprint sensor or face id sensor (as a form of 2FA or even just as a login straight up)